vllm/model_executor/models/qwen2_vl.py

```diff
@@ -631,25 +645,36 @@ def _get_max_image_info(
     image_processor,
     data_type_key: str = "image",
     mm_count: int = 1,
+    min_pixels: Optional[int] = None,
+    max_pixels: Optional[int] = None,
 ):
+    # Limit min / max pixels unless they're explicitly provided
+    if min_pixels is None:
+        min_pixels = max(image_processor.min_pixels, 28 * 28)
+    if max_pixels is None:
+        max_pixels = min(image_processor.max_pixels, 1280 * 28 * 28)
+
     return _get_vision_info(
         image_processor,
         height=9999999,
         width=9999999,
-
-        # Limit min / max pixels.
-        min_pixels=max(image_processor.min_pixels, 28 * 28),
```

Comment:
Just in case anyone knows - should this be enforced on actual requests also? I'm a bit confused why there are bounds on these values only while getting the max mm tokens, since it means actual requests can exceed the max 🤔

Reply:
> Should this be enforced on actual requests also?

You mean requests coming in from online inference? Probably not, since this is model specific.

Reply:
Not generically, I mostly meant in the input processor and multimodal mappers for this class. It seemed a bit weird to only use an upper bound on `max_pixels` in the max calculation, but not apply the same restriction in places like [this](https://github.com/vllm-project/vllm/blob/b47a739880b5240e1930e3bfe57cf8ab6ba234c2/vllm/model_executor/models/qwen2_vl.py#L830)

Reply:
Ah I see. @fyabc can you comment on this?

Reply:
Let's still merge this PR first as this issue is already present in the existing code.